test/e2e/framework/test_context.go

```diff
@@ -407,7 +407,7 @@ func createKubeConfig(clientCfg *restclient.Config) *clientcmdapi.Config {
 	return configCmd
 }
 
-func generateSecureToken(tokenLen int) (string, error) {
+func GenerateSecureToken(tokenLen int) (string, error) {
```

Comment:
If someone has an opinion on where to put this for everyone to consistently access it, or if I should just copy it into the remote runner code.

Reply:
Dunno if copying is ideal either. Maybe [util.go](https://github.com/kubernetes/kubernetes/blob/73190add9114a00ffe839acdd8c408c1e41ab4c5/test/e2e/framework/util.go)? I don't love kitchen sink files generally, but don't know if this one has any better home.

Reply:
It's also not the end of the world to leave it here for now.

Reply:
Need to add the comment of this function to pass pull-kubernetes-verify job.
```
Errors from golint:
test/e2e/framework/test_context.go:410:1: exported function GenerateSecureToken should have comment or be unexported
```